src/leo-server-app/src/server.ts

```diff
@@ -45,6 +45,8 @@ app.get("/", (req: Request, res: Response) => {
   res.send("Hello, this is Express + TypeScript");
```

Comment:
Do we need this here?

Reply:
base setup, i think its good just to have to see its running